app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedFragment.kt

```diff
@@ -47,6 +47,6 @@ class RecentlyPlayedFragment : InjectableFragment(), OngoingStoryClickListener {
   }
 
   override fun onOngoingStoryClicked(promotedStory: PromotedStory) {
```

Comment:
The function name should be updated too

Reply:
@anandwana001   in topic.proto  PromotedStoryList  already exists....there is also a ongoingStoryList...would you please tell me what to change in topic.proto

Reply:
@Prince-kushwaha Just update the function name in `PromotedStoryClickListener ` and everywhere else where it is used.